src/EditorConfigSamples.ruleset

```diff
@@ -0,0 +1,79 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Hmmmm. Maybe we should provide a `.ruleset` file at the root that conforms? [This](https://github.com/Dotnet-Boxed/Templates/blob/master/MinimumRecommendedRulesWithStyleCop.ruleset) is the one I use everywhere. It turns off document everything rules but generally sticks to defaults. If we do this, we should give it a nice name and description.

Reply:
**_Feel free to edit or delete this project._**  My goal was simply to remove warnings not related to the issue at hand.